plugins/svgtools/include/detray/plugins/svgtools/conversion/trajectory.hpp

```diff
@@ -0,0 +1,54 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s)
+#include "detray/intersection/detail/trajectories.hpp"
+#include "detray/plugins/svgtools/conversion/point.hpp"
+#include "detray/plugins/svgtools/meta/proto/trajectory.hpp"
+
+// System include(s)
+#include <vector>
+
+namespace detray::svgtools::conversion {
+
+/// @returns The proto trajectory of a vector of points.
+template <typename point3_t>
+inline auto trajectory(const std::vector<point3_t>& points) {
+    using p_trajectory_t = svgtools::meta::proto::trajectory<point3_t>;
+    p_trajectory_t p_trajectory;
+    p_trajectory._points = points;
+    return p_trajectory;
+}
+
+/// @returns The proto trajectory of a vector of a trajectory.
```

Comment:
Maybe smth like
```suggestion
/// @returns The proto trajectory of a parametrized trajectory.
```
?